src/package.lisp

```diff
@@ -75,7 +75,9 @@
 
    
    #:*user-1* #:*user-2* #:*user-3*
-   #:*tx-1* #:*tx-2* #:*tx-3*))
+   #:*tx-1* #:*tx-2* #:*tx-3*
+
+   #:eassert))
```

Comment:
@guitarvydas Why export this symbol?  Do you intend to make an API for others to use?